cafe/app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CartsController < ApplicationController
```

Comment:
cart cannot be service. We need to have clear idea what can be service and what cannot be.